### PR TITLE
Fix compilation error

### PR DESCRIPTION
**Pull request by justinfagnani**
- [ ] CHANGELOG.md has been updated

### CODE REVIEW

File: src/analyzer.ts [hunk 1]
```diff
@@ -94,9 +98,7 @@ export class Analyzer {
    * Note: this feature is experimental.
    */
   _fork(): Analyzer {
-    const analyzer = new Analyzer({});
```
Comment (justinfagnani) [inline src/analyzer.ts]: FYI, the error was that `{}` didn't match options because it doesn't have `urlLoader`

File: src/analyzer.ts [hunk 2]
```diff
@@ -48,8 +48,12 @@ export type LazyEdgeMap = Map<string, string[]>;
  */
 export class Analyzer {
   private _context: AnalysisContext;
-  constructor(options: Options) {
-    this._context = new AnalysisContext(options);
+  constructor(options: Options|AnalysisContext) {
+    if (options instanceof AnalysisContext) {
```
Comment (FredKSchott) [inline src/analyzer.ts]: use `options.constructor.name === 'AnalysisContext'` to work around different instances being passed around terrible npm trees.

Comment (justinfagnani) [inline src/analyzer.ts]: That doesn't trigger the type narrowing in TypeScript.

File: src/analyzer.ts (hunk 1 above)
Comment (FredKSchott) [inline src/analyzer.ts]: ugh silent build failures. Lets prioritize getting that fixed, want me to look into it or are you already?

**Pull request merged**